site/pages/what-we-do/header-slice/style.css

```diff
@@ -0,0 +1,120 @@
+@value badgerWhite, badgerRedOnWhite, badgerBlack from "../../../css/_colors.css";
+@value mediumScreen, largeScreen from "../../../css/_sizes.css";
+
+.pageHeaderContainer {
+  width: 280px;
+  margin: 0 auto;
+  display: flex;
+  composes: dividerWhite from "../../../css/_divider.css";
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  background-color: badgerWhite;
+  padding-top: 30px;
+  padding-bottom: 60px;
+}
+
+.pageHeaderContainer .left {
+  text-align: left;
+  margin-right: auto;
+}
+
+.pageHeaderContainer .right {
+  text-align: right;
+  margin-left: auto;
+}
```

Comment:
Don't think the descendant selector is needed here (ie. it should be specific enough)?